docs/source/gbd2017_models/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -464,7 +505,14 @@ on the methodology described here.
 
 .. todo::
 
-	Describe how to handle the changing prev_ira/prev_total_anemia across age groups.
+	Describe how to handle the changing prev_ira/prev_total_anemia across age 
+	groups, which generally decreases by about 2% between the early NN and 1-4 
```

Comment:
Similarly, it would be good to cite where you got this figure of 2% - either from your own code or from GBD compare.